### DISCUSSION TITLE
How to find a 95% confidence interval for the steady state probability of being in either of several states.

### DISCUSSION THREAD

**Discussion by YusukeGoto510** [bmcclintock/momentuHMM#145] [Q&A] [answered]
Hello!
I have classified a time series of a variable “x” with a covariant “y” into 3 states.  I learned that the 95% CI of the stationary state probability of being in each of 3 states can be obtained by plotstationary(fit,plotCI=TRUE) or CIreal() functions, but I wonder if is it possible to find the 95% CI of the stationary state probability of being in “either" state 1 or 2? 
Thank you in advance.

**Comment by bmcclintock** [comment, marked as answer]
I think the best you could do using `momentuHMM` output is to sum together the stationary probabilities for states 1 and 2 $(\pi_{12}=\pi_1+\pi_2)$, derive the standard error as $\text{SE}_{12}=\sqrt{\text{SE}_1^2+\text{SE}_2^2}$, and then calculate the 95% CI using a logit transform (to keep the CI bound between 0 and 1):

$\text{CI} = \frac{1}{1 + \exp(-(\log(\pi_{12}/(1-\pi_{12})) \pm  q_{1-\alpha/2} \left(1/(\pi_{12}-\pi_{12}^2)\right) \text{SE}_{12}))}$

where $q_{1-\alpha/2}$ is the standard normal quantile for the $\alpha$ level of significance.
